src/HSE/All.hs

```diff
@@ -63,12 +63,14 @@ data ParseMode = ParseMode {
         }
 
 defaultParseMode :: ParseMode
-defaultParseMode = ParseMode {
-        baseLanguage = Haskell2010,
-        extensions = [],
-        fixities = Just preludeFixities
-        }
-
+defaultParseMode =
+  ParseMode {
+      baseLanguage = Haskell2010
+    , extensions = []
+    , fixities = Just preludeFixities
+    }
+  where
+    preludeFixities = map toHseFixity $ concatMap (fromFixitySig . mkFixitySig) GhclibParserEx.preludeFixities
```

Comment:
Can we have one function that goes from `(String, Fixity) -> FixityInfo` instead of roundtripping through FixitySig.

Reply:
You got it.